website/docs/install-config/install/docker-compose.mdx

```diff
@@ -117,6 +117,10 @@ The `docker-compose.yml` file statically references the latest version available
 
 To start the initial setup, navigate to `http://<your server's IP or hostname>:9000/if/flow/initial-setup/`.
 
+:::info
+User will get a `404 NOT FOUND` if initial setup URL doesn't include the trailing forward slash `/`. Make sure you use the complete url (`http://<your server's IP or hostname>:9000/if/flow/initial-setup/`) including the trailing forward slash.
```

Comment:
Let's reword to avoid that ugly `404 NOT FOUND` wording.

Reply:
Hah, sure, what did you have in mind @rissson? `404`s are by nature ugly. Are you proposing that we don;t menetion the possibility of a 404 at all, and say something like "The authentik UI will not load if the initial setup URL doesn't include the trailing forward slash `/`."

Because a 404 is indeed the most likely result, it's good to be upfront and honest. ;-)

Reply:
![image](https://github.com/user-attachments/assets/548e44da-4754-4c89-838d-90dbe340b323)
this is the actual error, just `Not found`, not `404 NOT FOUND`.

This:

![image](https://github.com/user-attachments/assets/1c555e0a-ec7b-477d-8be4-5e282e1444ae)

looks much better than this imo, and is more accurate:

![image](https://github.com/user-attachments/assets/3e0a7b55-5f60-469e-9d85-a99a42f805e6)

Reply:
done, @rissson.